spharpy/samplings/coordinates.py

```diff
@@ -8,7 +8,7 @@ class SamplingSphere(pf.Coordinates):
 
     def __init__(
             self, x=None, y=None, z=None, n_max=None, weights: np.array = None,
-            comment: str = ""):
+            comment: str = "", radius_tolerance=1e-6):
```

Comment:
and now we need to add it for all from* methods, because right now we cannot set it via e.g. ``SamplingSphere.from_spherical_elevation(radius_tolerance=.1)``

this could be also done in an other pr together with quadrature attribute.

Reply:
Done it directly here